src/wallets/browser/NearWallet.ts

```diff
@@ -17,6 +17,9 @@ import {
 } from "../../interfaces/IWallet";
 import ProviderService from "../../services/provider/ProviderService";
 import getConfig from "../../config";
+import { Logger } from "../../services/logging.service";
+
+const logger = new Logger();
```

Comment:
We could remove this import and new up by using a default export:

```ts
// logging.service.ts
export class Logger { /* ... */ }

export default new Logger();

// ---------------------

// module-that-wants-to-use-logger.ts
import logger from ".../services/logging.service";

logger.log("Hello World!");

// ---------------------

// logging.service.spec.ts
import { Logger } from "./logging.service";

beforeEach(() => {
  logger = new Logger();
  logger.debug = true;
});

afterEach(() => {
  logger.debug = false;
});

// ....
```

Reply:
The reason I left it really was to be able to turn on a new logger instance outside the project and use a custom logging method like sentry to get logs from this module. That is the reason I would not instanciate in the same file. You could just import beforehand the Logger and then instanciate the `NearWalletSelector` library.